api/build.gradle

```diff
@@ -180,6 +203,11 @@ test {
   }
 }
 
+integration {
+  // These tests should always run when requested since they consume and produce side-effects.
```

Comment:
What are the side effects of your current integration test? (I thought it wasn't actually writing data...)

Reply:
None produced yet (you are correct). Only consumed.